### ISSUE TITLE
There are some questions about the loss function

### ISSUE THREAD

**Issue by AdaAkanishi** [GSNN/GSNN#1]
Hello!Congratulations on your paper has been accepted by nips 2020！
Although I'd like to know the idea you mentioned，I didn't find your article online . Having been read the source code，I still have some question about the code. In "GSNNTrain.py", "getloss" function have 4 return values: reconstruct_total, kl_total, q_yl, kl_pq ,could you tell me the meaning about them?And why loss = reconstruct_total + kl_total + kl_pq + lamda * q_yl ?

**Comment by chenxinst** [comment]
We adopt the amortised variational inference to approximate the intractable joint posterior for missing labels and the latent variable z. We use three neural networks to  instantiate the distribution q_phi and p_theta, and optimize all parameters by maximizing the ELBO function. More details can be found in our paper, which will be avaliable on arXiv soon.

**Comment by chenxinst** [comment]
We adopt the amortised variational inference to approximate the intractable joint posterior for missing labels and the latent variable z. We use three neural networks to  instantiate the distribution q_phi and p_theta, and optimize all parameters by maximizing the ELBO function. More details can be found in our paper, which will be avaliable online soon.
